test/helpers/kubectl.go

```diff
@@ -2337,12 +2337,18 @@ func (kub *Kubectl) CiliumCheckReport(ctx context.Context) {
 // `deadlocks` or `segmentation faults` messages. In case of any of these
 // messages, it'll mark the test as failed.
 func (kub *Kubectl) ValidateNoErrorsInLogs(duration time.Duration) {
+	kub.ValidateNoErrorsInLogsWhitelist(duration, map[string]bool{})
```

Comment:
You should be able to pass in the whitelist as a nil. It's not a big deal. We have every combination of style in the code already.